bears/general/TextLintBear.py

```diff
@@ -0,0 +1,238 @@
+            This rule does not allow to create an empty section.
+            For example, there is an empty section ``# Header A`` below::
+
+                # Header A
+
+                # Header B
+                Text.
+
+        :param check_date_weekday_mismatch:
+            This rule finds a mismatch between a date and the corresponding
+            weekday.
+        :param check_grammar:
+            This rule checks your English grammar with Ginger Proofreading.
+        :param max_lines_per_file:
+            Number of lines allowed per file.
+        :param max_comma_per_sentence:
+            Number of commas allowed per sentence.
+        :param no_good_words:
+            Set of NG (No Good) words to check for.
+        :param period_in_list_item:
+            Checks whether a sentence in a list item ends with a period.
+        :param minimum_acronym_length:
+            Minimum length for unexpanded acronyms.
+        :param maximum_acronym_length:
+            Maximum length for unexpanded acronyms.
+        :param ignore_acronyms:
+            A list that contains the acronyms to ignore.
+        :param check_with_rousseau:
+            This rule checks English writing using Rousseau, which is a
+            lightweight proofreader written in JavaScript.
+            It can check:
+            - Passive voice
+            - Lexical illusions – cases where a word is repeated
+            - 'So' at the beginning of the sentence
+            - Adverbs that can weaken meaning: really, very, extremely, etc.
+            - Readability of sentences
+            - Simpler expressions
+            - Weasel words
+            - If a sentence is preceded by a space
+            - If there is no space between a sentence and its ending
+              punctuation
+            - If sentences are starting with uppercase letter
+        :param check_with_alex:
+            This rule helps you find gender favouring, polarising, race
+            related, religion inconsiderate, or other unequal phrasing
+            and checks for:
+            - Gendered work-titles, for example warning about ``garbageman``
+              and suggesting ``garbage collector`` instead
+            - Gendered proverbs, such as warning about ``like a man`` and
+              suggesting ``bravely`` instead, or warning about ``ladylike``
+              and suggesting ``courteous``
+            - Blunt phrases, such as warning about ``cripple`` and suggesting
+              ``person with a limp`` instead
+            - Intolerant phrasing, such as warning about using ``master`` and
+              ``slave`` together, and suggesting ``primary`` and ``replica``
+              instead
+        :param check_common_misspellings:
+            This rule helps to find common misspellings from Wikipedia's
+            list of common misspellings.
+        :param allow_passive_voice:
+            Allows passive voice.
+        :param allow_so_beginning:
```

Comment:
I've followed the standard used in https://github.com/coala/coala-bears/blob/master/bears/natural_language/WriteGoodLintBear.py#L37.

Maybe we should rename both the instances or keep this as it is ?